snuba/snuba_migrations/replays/0014_add_id_event_columns.py

```diff
@@ -0,0 +1,43 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import UUID, Column
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.operations import OperationTarget
+
+LOG_LEVELS = ["fatal", "error", "warning", "info", "debug"]
```

Comment:
An alternative idea for this schema.  Instead of creating multiple columns for each log level we should instead add a single `event_id` column and a `event_log_level` column?  `event_log_level` encoded as low-cardinality.

When you query for an `event_id` you target a singular column without needing to find out which log-level it is first.  When you count different log-levels you target the `event_log_level` column with a `countIf` function.  If in the future you want to support more log levels it does not require a migration.

Reply:
i don't think we'd ever want to support more log levels. we've had the same 5 hard coded in sentry for forever. https://develop.sentry.dev/sdk/event-payloads/types/